src/imitation/algorithms/adversarial/airl.py

```diff
@@ -69,7 +70,7 @@ def logits_expert_is_high(
         action: th.Tensor,
         next_state: th.Tensor,
         done: th.Tensor,
-        log_policy_act_prob: th.Tensor,
+        log_policy_act_prob: Optional[th.Tensor] = None,
```

Comment:
lol I'm amazed pytype did not catch this, very sad

Reply:
I think pytype lets you replace the supertype's function signature in the subclass without complaining, I saw a bunch of other errors like this by mypy that pytype never flagged. (Even though many times the only difference was the variable names, not the types. But this is still a problem if you call them as keyword arguments. We didn't, hence why there were no errors. But still better to be consistent).